pkgs/by-name/ma/mailsy/package.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildNpmPackage, fetchFromGitHub, }:
```

I think you used the wrong formatter. The nixfmt you should use is [nixfmt-rfc-style](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=nixfmt-rfc-style)

```suggestion
{
  lib,
  buildNpmPackage,
  fetchFromGitHub,
}:
```